src/transformStateWithClones.js

```diff
@@ -8,6 +8,51 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const ACTION_ADD = 'addProperties';
+  const ACTION_REM = 'removeProperties';
+  const ACTION_CLR = 'clear';
+  const copyState = [];
+  let obj = { ...state };
+
+  for (let i = 0; i < actions.length; ++i) {
+    const curOper = actions[i].type;
+
+    if (i > 0) {
+      obj = { ...copyState[i - 1] };
+    }
```

Comment:
looks redundant

Reply:
I have no idea how to fix it now. But I agree with you.